manager/status.py

```diff
@@ -0,0 +1,136 @@
+                    usage.used / 1024 / 1024, 6
+                ),  # Convert from bytes to MBytes
+                "UsedPercent": usage.percent,
+            }
+            self.payload["Metrics"]["Disks"].append(disk_data)
+
+    def add_processes(self):
+        for proc in psutil.process_iter(["pid", "name", "cmdline", "memory_info"]):
+            MemoryUsedMBytes = ""
+            # Convert from bytes to MBytes
+            if proc.info["memory_info"] is not None:
+                MemoryUsedMBytes = round(proc.info["memory_info"].rss / 1024 / 1024, 6)
+
+            # cmd line
+            cmdLine = ""
+            if proc.info["cmdline"] is not None:
+                cmdLine = " ".join(proc.info["cmdline"])
+                cmdLine = (cmdLine[:75] + "...") if len(cmdLine) > 75 else cmdLine
+            process_data = {
+                "Name": proc.info["name"],
+                "Command": cmdLine,
+                "MemoryUsedMbytes": MemoryUsedMBytes,
+                "PID": proc.info["pid"],
+                "ParentPID": proc.ppid(),  # Get parent PID
+            }
+            self.payload["Metrics"]["Processes"].append(process_data)
+
+    def add_docker_services(self):
+        client = docker.from_env()
+        for container in client.containers.list():
+            container_attrs = container.attrs
+            docker_service_data = {
+                "Name": container.name,
+                "Image": container.image.tags[0] if container.image.tags else "",
+                "Command": " ".join(container.attrs["Config"]["Cmd"]),
+                "Environment": {
+                    item.split("=")[0]: item.split("=")[1]
+                    for item in container.attrs["Config"]["Env"]
+                },
+                "CreatedAt": container_attrs["Created"],
+                "ExitedAt": container_attrs["State"]["FinishedAt"],
+                "Status": container_attrs["State"]["Status"],
+                "Running": container_attrs["State"]["Running"],
+                "Paused": container_attrs["State"]["Paused"],
+                "Restarting": container_attrs["State"]["Restarting"],
+                "OOMKilled": container_attrs["State"]["OOMKilled"],
+                "Dead": container_attrs["State"]["Dead"],
+                "Pid": container_attrs["State"]["Pid"],
+                "ExitCode": container_attrs["State"]["ExitCode"],
+                "Error": container_attrs["State"]["Error"],
+            }
+            self.payload["Metrics"]["docker-services"].append(docker_service_data)
+
+    def get(self):
+        usedMB = round(self.payload["Metrics"]["MemoryUsedMBytes"])
+        cpu = self.payload["Metrics"]["CPULoadPercent"]
+        self.status = f"RAM = {usedMB}mb, CPU = {cpu}%"
+        # self.error = ""
+        return {
+            "Timestamp": self.timestamp,
+            "Status": self.status,
+            "Error": self.error,
+            "Payload": self.payload,
+        }
+
+
+# Usage
```

Comment:
Remove these?

Reply:
i like to keep it for sake of debug, but not crucial